src/com/tapia/foss/ds/List.java

```diff
@@ -17,6 +17,7 @@
 package com.tapia.foss.ds;
```

Comment:
The changes in List.java are unrelated to multiple selection bug. Please submit a separate commit for it and remove it from the current commit.